### ISSUE TITLE
A word of warning to all- Moodstocks 4 SDK no longer supported

### ISSUE THREAD

**Issue by remib** [thomasforth/MS4Plugin#11]
Having completed our App using this plugin and distributing to the App Stores, our client attempted to purchase a Moodstocks subscription to increase the image allowance. We were informed that Moodstocks are no longer supporting their legacy product and they're focused on their new product which is now available in Alpha https://moodstocks.com/.

Annoyingly, the Moodstocks website was still accepting signups for their legacy product up until very recently and there was no mention of the discontinuation of their legacy product. We are in talks with Moodstocks to come to an agreement.

As a potential fallback, there is now a working cordova plugin for Vuforia https://github.com/thisisbd/cordova-plugin-vuforia. I have tested this myself and can confirm it works on both iOS and Android. The only caveat is it does not support cloud databases so image targets must be stored on-device, meaning an App update is required to update image recognition targets.

**Comment by hmichelkernix** [comment]
Hello @remib thank you for this news.

**Comment by Juliobh** [comment]
Hi @remib!
Any feedback from moodstocks team? We are with the same problem... Our app is almost ready and we're worried about your comment.

We sent an email today asking them about.

Thanks!

**Comment by remib** [comment]
@Juliobh Nope, they continue to be unresponsive to any form of communication. We have had to recreate the App using the Vuforia and the plugin available here https://github.com/mattrayner/cordova-plugin-vuforia however it's far from perfect and only allows for on-device image targets, no cloud support. 

If you get any response from Moodstock could you let me know please?

**Comment by remib** [comment]
The plot thickens; Google have bought Moodstocks. I don't know what implications this will have going forward but I hope they keep the platform open for others to use.

http://venturebeat.com/2016/07/06/google-buys-machine-learning-startup-moodstock-to-help-your-phones-camera-identify-objects/
